woocommerce/woocommerce
# Update sample_products.xml to include attachments

### All Submissions:






### Changes proposed in this Pull Request:

The XML import file included with the sample data current does not include attachments which when imported results in no images. This PR include an updated xml import file with the all the required data to result in a successful product import.

Closes #21663

### How to test the changes in this Pull Request:

1. Start with a clean WP installations with WooCommerce plugin and no products or images imported previously.
2. Import products via the WP Importer, Tools -> Import
3. Ensure that images are imported and displaying.

### Other information:




### Changelog entry

> Fix - XML Sample Data import file was missing images, not has all the required data.
